sdk/python/tests/expediagroup/test_milvus_online_store.py

```diff
@@ -1,37 +1,346 @@
+                )
             ],
-            "auto_id": True,
-        }
+            entities_to_delete=None,
+            entities_to_keep=None,
+            partial=None,
+        )
 
-        milvus.create_collection(collection_name, fields)
+        print(utility.list_collections())
+        # Here we want to open and check whether the collection was added and then close the connection.
+        with PymilvusConnectionContext():
+            assert utility.has_collection(self.collection_to_write) is True
 
-    collection = milvus.describe_collection(collection_name)
-    assert collection.get("collection_name") == collection_name
+    def test_milvus_update_add_existing_collection(
+        self, repo_config, caplog, milvus_online_setup
+    ):
 
-    online_store_creator.teardown()
+        self.setup_method(milvus_online_setup)
+        # Creating a common schema for collection
+        schema_vector = [
+            Field(
+                name="feature1",
+                dtype=Array(Float32),
+                tags={
+                    "is_primary": "False",
+                    "description": "float32",
+                    "dimension": "128",
+                },
+            ),
+            Field(
+                name="feature2",
+                dtype=Int64,
+                tags={"is_primary": "True", "description": "int64"},
+            ),
+        ]
+
+        # Creating a common schema for collection to directly add to Milvus
+        schema = CollectionSchema(
+            fields=[
+                FieldSchema(
+                    "int64", DataType.INT64, description="int64", is_primary=True
+                ),
+                FieldSchema(
+                    "float_vector", DataType.FLOAT_VECTOR, is_primary=False, dim=128
+                ),
+            ]
+        )
+
+        # Here we want to open and add a collection using pymilvus directly and close the connection.
+        with PymilvusConnectionContext():
+            Collection(name=self.collection_to_write, schema=schema)
+            assert utility.has_collection(self.collection_to_write) is True
+            assert len(utility.list_collections()) == 1
+
+        MilvusOnlineStore().update(
+            config=repo_config,
+            tables_to_delete=[],
+            tables_to_keep=[
+                VectorFeatureView(
+                    name=self.collection_to_write,
+                    schema=schema_vector,
+                    source=SOURCE,
+                    vector_field=VECTOR_FIELD,
+                    dimensions=DIMENSIONS,
+                    index_algorithm=INDEX_ALGO,
+                )
+            ],
+            entities_to_delete=None,
+            entities_to_keep=None,
+            partial=None,
+        )
+
+        # Here we want to open and add a collection using pymilvus directly and close the connection, we need to check if the collection count remains 1 and exists.
+        with PymilvusConnectionContext():
```

Comment:
does the embedded Milvus actually persist data after it is stopped?

Reply:
Yes it does